libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -137,6 +144,10 @@ impl<'a> JsiiImporter<'a> {
 					"map" => self.wing_types.add_type(Type::Map(wing_type)),
 					_ => panic!("Unsupported collection kind '{}'", collection_kind),
 				}
+			} else if let Some(Value::Object(_)) = obj.get("union") {
+				// Wing does not support union types, so we'll model it the same way as if
+				// we saw an "any" in a JSII library
```

Comment:
Do we have a mechanism to emit compiler warnings/info?
```suggestion
				// Currently Wing does not support union types (see https://github.com/winglang/wing/issues/977), 
				// so we'll model it the same way as if we saw an "any" in a JSII library.
```

Reply:
https://github.com/winglang/wing/issues/1475